src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Cat;
+import core.basesyntax.model.Dog;
+import java.time.LocalDate;
+
+public class HelloWorld {
+    private String str;
```

Comment:
is this variable informative?

Reply:
It might be "currentYear" instead of "variable", for example.